app/src/main/java/org/oppia/app/utility/SplitScreenManagerModule.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import dagger.Module
+import dagger.Provides
+import org.oppia.app.application.ApplicationContext
+
+/** A module for [SplitScreenManager] for dagger injection. */
+@Module
+class SplitScreenManagerModule {
+
+  @Provides
+  fun provideSplitScreenManager(@ApplicationContext context: Context): SplitScreenManager {
+    return SplitScreenManager(context)
```

Comment:
This provider is unnecessary--``SplitScreenManager`` is already injectable, so its type is automatically bound.

Reply:
Removed

Reply:
Resolving this on behalf of Ben.